### DISCUSSION TITLE
Error Code 504

### DISCUSSION THREAD

**Discussion by adhiduzz** [spencerwooo/onedrive-vercel-index#615] [Q&A] [answered]
Hi

i got error 504, and nothing further info

![error504](https://user-images.githubusercontent.com/104109164/164345143-55f6c62c-c6f4-43d9-b7df-03e0ecbb3c32.png)

**Comment by spencerwooo** [comment, marked as answer]
Check your Vercel function logs for error details. I should assume it has something to do with your `REDIS_URL` - so double check that with the documentation.

**Reply by xiaoRran** [reply]
![](https://sady-1252199732.cos.ap-nanjing.myqcloud.com/202204211428167.png)
这是怎么回事

**Reply by myl7** [reply]
Redis 似乎没有连接成功。能确保你的 Redis URL 是正确的吗？（因为普通的、带 TLS 的 Redis URL 应该是以 `rediss://:` 开头的）

**Reply by adhiduzz** [reply]
thank you, yes you're right..issue come from redis_url,,
it just lack "s" in rediss 😂
